Week-1/Exercise1.2/src/index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<head>
+    <meta charset="UTF-8">
+    <title>Pesto Playlist</title>
+    <link rel="stylesheet" href="index.css">
```

Comment:
Understand header section better in the perspective of performance and browser rendering